library/oneview_scope_facts.py

```diff
@@ -16,20 +16,17 @@
 # limitations under the License.
```

Comment:
The `# -*- coding: utf-8 -*-` wasn't included in this file.
Also missing update the (2016-2017)